guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,352 @@
+            Party
+         </th>
+         <th>
+            Requirements
+         </th>
+         <th>
+            Benefits
+         </th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Sender</td>
+         <td>
+            <ul>
+               <li>Wallet support</li>
+               <li>Internet connection</li>
+            </ul>
+         </td>
+         <td>
+            <ul>
+               <li>Improved Privacy</li>
+            </ul>
+         </td>
+      </tr>
+      <tr>
+         <td>Receiver</td>
+         <td>
+            <ul>
+               <li>Hot wallet (for signing transactions)</li>
+               <li>Serve/maintain secure web endpoint</li>
+               <li>Wallet support</li>
+               <li>Funds in hot wallet</li>
+            </ul>
+         </td>
+         <td>
+            <ul>
+               <li>Privacy</li>
+               <li>UTXO consolidation</li>
+               <li>Fee savings</li>
+            </ul>
+         </td>
+      </tr>
+   </tbody>
+</table>
+</div>
+
+
+### Sending Payjoins
+
+Participating in payjoin transactions is relatively straightforward for a sender. They only need a supporting wallet and have to be online during the process.
+
+
+#### Investigation: Blue Wallet [Video](https://www.youtube.com/watch?v=xowwF2yFSZ8) Breakdown
+
+We analyzed the payjoin user flow in BlueWallet. Read the detailed investigation [here](https://docs.google.com/document/d/1_de2pkMREGpZQwOefQdH-MZNSUPazObgRK1cruiWBmo/edit#bookmark=id.jamlvi37p922).
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/payjoin-case-study/bluewallet-sender-flow.png"
+   retina = "assets/images/guide/case-studies/payjoin-case-study/bluewallet-sender-flow@2x.png"
+   mobile = "assets/images/guide/case-studies/payjoin-case-study/bluewallet-sender-flow-mobile.png"
+   mobileRetina = "assets/images/guide/case-studies/payjoin-case-study/bluewallet-sender-flow-mobile@2x.png"
+   alt-text = "bluewallet sender flow"
+   width = 800
+   height = 54
+%}
+
+Here are the findings:
+
+- The fee rate drops once transaction size increases during the payjoin process,  which may change estimated confirmation time
+- The flow results in a transaction that has a round amount fee-rate after removing one input, which may be used by chain analysis firms to spot payjoins
+
+#### The Payjoin Sender Flow
+
+We designed a sender flow that attempts to address the issues identified above.
+
+For the purpose of creating the sender flow, we will assume that the receiver only contributes UTXOs but does not contribute to the fees since this allows a simple, automated process for both parties.
+
+### In short, the sender flow outlined here asks the user to choose a fee-range instead of a fee-amount (or fee-rate) while keeping the rest of the user flow almost exactly the same. We use it to set 3 optional parameters specified in BIP-78, which can be used to construct a simple but effective payjoin implementation.
```

Comment:
```suggestion
** In short, the sender flow outlined here asks the user to choose a fee-range instead of a fee-amount (or fee-rate) while keeping the rest of the user flow almost exactly the same. We use it to set 3 optional parameters specified in BIP-78, which can be used to construct a simple but effective payjoin implementation.**
```